lib/portlayer/exec/base.go

```diff
@@ -36,6 +37,9 @@ import (
 	log "github.com/Sirupsen/logrus"
 )
 
+// vSphere error msg when sending the kill signal to a containerVM that is already powered off
+const opNotPerformedPoweredOff = "The attempted operation cannot be performed in the current state (Powered off)"
```

Comment:
I believe this error message is constructed from lower down, and potentially contains a fault type. Looking.

Reply:
I think the error msg is constructed in a way similar to this https://github.com/vmware/vic/blob/master/vendor/github.com/vmware/govmomi/simulator/simulator.go#L94. And the msg itself comes from the http response body of `startGuestProgram()`. I suspect if we can find the error definition.

Reply:
According to @dougm , this is the `invalidPowerState` error.

Reply:
Yeah I saw that as well, looks like doug beat me to it :) thanks @chengwang86